### PR TITLE
new: add a property to tell which identity the recipes will attempt t…

### PR DESCRIPTION
**Pull request by t00f**
Adding a new property `targetIdentities` so we can check what are the authorizations the user will need to have to access this recipe.

### CODE REVIEW

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "add_recipe_targetIdentities",
    "component": "backend",
    "pr-id": "421",
    "commit-sha": "ae2fd93b0b5de4c51dec879d8eaedeee26fc4c89"
  },
  {
    "project": "add_recipe_targetIdentities",
    "component": "gaia",
    "pr-id": "335",
    "commit-sha": "1ebe62e0971094bb41f23cbfcea23a229534c719"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "add_recipe_targetIdentities",
    "component": "backend",
    "pr-id": "421",
    "commit-sha": "9ad2b4131bcb87a8192aee93dbb023699b9a135e"
  },
  {
    "project": "add_recipe_targetIdentities",
    "component": "gaia",
    "pr-id": "335",
    "commit-sha": "1ebe62e0971094bb41f23cbfcea23a229534c719"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "add_recipe_targetIdentities",
    "component": "backend",
    "pr-id": "421",
    "commit-sha": "9ad2b4131bcb87a8192aee93dbb023699b9a135e"
  },
  {
    "project": "add_recipe_targetIdentities",
    "component": "gaia",
    "pr-id": "335",
    "commit-sha": "8dbe92ee4f5f9ed0e811c78c8d9647e3ba821daf"
  }
]
```

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "add_recipe_targetIdentities",
    "component": "backend",
    "pr-id": "421",
    "commit-sha": "db61fbd69a4eced929332b7a8835550e8e32f62a"
  },
  {
    "project": "add_recipe_targetIdentities",
    "component": "gaia",
    "pr-id": "335",
    "commit-sha": "8dbe92ee4f5f9ed0e811c78c8d9647e3ba821daf"
  }
]
```

File: specs/recipe.spec [hunk 1]
```diff
@@ -88,6 +88,17 @@ attributes:
     exposed: true
     stored: true
 
+  - name: targetIdentities
```
Comment (primalmotion) [inline specs/recipe.spec]: Not stored?

**Comment by t00f** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "add_recipe_targetIdentities",
    "component": "gaia",
    "pr-id": "335",
    "commit-sha": "0de1a906dd64be32f0863dcc26d318e1f423e20b"
  },
  {
    "project": "add_recipe_targetIdentities",
    "component": "backend",
    "pr-id": "421",
    "commit-sha": "0d831a364f55f963a535723ad75775243e8ed915"
  }
]
```

**Pull request merged**